demo/main.py

```diff
@@ -1,36 +1,38 @@
 import os
 import sys
 
-sys.path.append(os.path.abspath(os.path.dirname(__file__)).rsplit("akivymd", 1)[0])
+sys.path.append(os.path.dirname(__file__).split("akivymd")[0])
```

Comment:
assume my demo path is `D:\python\akivymd_component\akivymd\demo>` then it will append `D:\python\` instead of `D:\python\akivymd_component\` to `sys.path` which is wrong and will cause `no module named 'akivymd' ` error